sql/pre_install/schemas.sql

```diff
@@ -6,4 +6,7 @@ CREATE SCHEMA IF NOT EXISTS _timescaledb_catalog;
 CREATE SCHEMA IF NOT EXISTS _timescaledb_internal;
 CREATE SCHEMA IF NOT EXISTS _timescaledb_cache;
 CREATE SCHEMA IF NOT EXISTS _timescaledb_config;
+CREATE SCHEMA IF NOT EXISTS timescaledb_experimental;
 GRANT USAGE ON SCHEMA _timescaledb_cache, _timescaledb_catalog, _timescaledb_internal, _timescaledb_config TO PUBLIC;
+GRANT USAGE ON SCHEMA timescaledb_experimental TO PUBLIC;
```

Comment:
Why not the use of underscore as in _timescaledb_experimental?

Reply:
To be aligned with out the analytics extension names their experimental schema. It is also not necessary since this is a publicly available schema and not anything internal, similar to `timescaledb_information`.

Reply:
Ok, sounds good